.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for concrete
+title: '[FeatureReq]'
+labels: ''
```

Comment:
Maybe we could add a feature request label as well ?

Reply:
Good Idea, And remove the title scheme then?